### ISSUE TITLE
Rebasing CRLF vs LF issue

### ISSUE THREAD

**Issue by MasterJames** [mrdoob/three.js#8632]
I've been tormented by my Editor, GitHub desktop & website on Win and git
Still unclear on best practices/settings here.
Anyway
https://github.com/mrdoob/three.js/blob/dev/docs/api/loaders/ImageLoader.html
and TextureLoader.html 
Have only LF and this I think has been causing me some problems.
I've always hated Windows people for adding Carriage Return dodo to new lines.
Still I've been messing with
git config --global core.autocrlf
and core.eol 
and feeling no love for all this. Any advice would be helpful.

Basically those 2 example files are not consistent with the others in the bunch and maybe causing some trouble for others too.
What is it that they are suppose to be? CRLF for HTML and LF for JS
Maybe GitHub has (or should have) a way to convert all always?

...

##### Three.js version

- [x] Current Dev Branch

**Comment by MasterJames** [comment]
I've had to set this to false
git config --global core.autocrlf false
then when I commit & push it warns to fix above value to true, but I say NO commit anyways.
Then the compare resolves to no change identical.
All my HTML files have in three.js\docs\api\extras\helpers have only LF but the main dev branch they are not.

**Comment by MasterJames** [comment]
Well I finally got my main dev fork purified of those horrible new line issues and all changes so I can rebase and work with the recommend patch route.
https://github.com/mrdoob/three.js/issues/8632
